pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -664,6 +667,51 @@ public static class QueryOptionValue {
       }
     }
 
+    /**
+     * Calcite and Pinot rule names / descriptions
+     * used for enable and disabling of rules, this will be iterated through in PlannerContext
+     * to check if rule is disabled.
+     */
+    public static class PlannerRuleNames {
+      public static final String FILTER_INTO_JOIN = "FilterIntoJoinRule";
+      public static final String FILTER_AGGREGATE_TRANSPOSE = "FilterAggregateTransposeRule";
+      public static final String FILTER_SET_OP_TRANSPOSE = "FilterSetOpTransposeRule";
+      public static final String PINOT_PROJECT_JOIN_TRANSPOSE = "ProjectJoinTransposeRule";
```

Comment:
(minor) I'd suggest removing the `PINOT` from the key so that key and value can match. In the future if we need to modify a Calcite rule, we won't need to also modify these constants